tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1240,10 +1240,12 @@ namespace TensileLite
         if(internalArgsSupport.useUniversalArgs)
         {
             auto defaultWGM = sizeMapping.workGroupMapping;
-            if(sizeMapping.streamK != 0)
+            // Allow Origami dynamic WGM mapping for all kernels (enabled by setting TENSILE_STREAMK_DYNAMIC_WGM)
```

Comment:
Why remove if(sizeMapping.streamK != 0) here? Is it not necessary?

Reply:
I think before it only worked for where SK != 0, i.e. the non-Stream-K kernel could not use DynamicWGM. Now this enables it for all kernels, even the ones that do not set Stream-K as enabled.